calcom/cal.com
# feat: Plain integration

## What does this PR do?



This is a Cal & Plain.com integration that allows us to display customer cards on plain.com to speed up the process of helping them.

- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)



will send one privately, includes env file

## Mandatory Tasks (DO NOT REMOVE)

- [y] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [y] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [y] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?



- Are there environment variables that should be set?
         yes: PLAIN_API_KEY, PLAIN_API_URL, HMAC_SECRET_KEY
- What are the minimal test data to have? 
         users in cal.com local environment
- What is expected (happy path) to have (input and output)? 
        you can see the user card in plain.com
- Any other important info that could help to test that PR

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings
